docs/README.md

```diff
@@ -211,7 +211,7 @@ The following versioned modules are provided by dcrd repository:
 * [certgen](https://github.com/decred/dcrd/tree/master/certgen) - Provides a
   function for creating a new TLS certificate key pair, typically used for
   encrypting RPC and websocket communications
-* [addrmgr/v2](https://github.com/decred/dcrd/tree/master/addrmgr) - Provides a
+* [addrmgr/v3](https://github.com/decred/dcrd/tree/master/addrmgr) - Provides a
```

Comment:
Please don't update this yet. We have to wait until a new version is tagged to do it. Otherwise, people looking at the documentation will think they can get the v3 module and they won't be able to because it won't be tagged yet.

The only reason we're able to access it before it's released internally in the repo is due to the replace.